### DISCUSSION TITLE
Default bed view

### DISCUSSION THREAD

**Discussion by maziggy** [supermerill/SuperSlicer#2215] [General]
Is there a way to configure the default bed view? For example like seen in the attached screenshot.
<img width="1542" alt="Screenshot 2022-01-13 at 17 35 58" src="https://user-images.githubusercontent.com/17797632/149370898-5ac3ecdd-57b1-47b9-a275-a95989c05f24.png">

Thx, Martin

**Comment by neophyl** [comment]
Not that I’ve seen. You know about the shortcuts though ?  You can use the number keys to switch to various views. They are listed in the help.

**Comment by maziggy** [comment]
The 1 is my friend :) Thanks.
